src/Sulu/Bundle/MediaBundle/Resources/config/ffmpeg.xml

```diff
@@ -23,5 +23,11 @@
             </argument>
             <argument id="logger" type="service" on-invalid="null"/>
         </service>
+
+        <service id="sulu_media.video_properties_provider" class="Sulu\Bundle\MediaBundle\Media\PropertiesProvider\VideoPropertiesProvider">
+            <argument type="service" id="sulu_media.ffprobe"/>
+
+            <tag name="sulu_media.media_properties_provider"/>
```

Comment:
think i would call this just `properties_provider` because the interface is called `PropertiesProviderInterface `

```suggestion
            <tag name="sulu_media.properties_provider"/>
```

Reply:
I did use the `MediaPropertiesProviderInterface` now so I think I would keep here `sulu_media.media_properties_provider`.

Reply:
sounds good 🙂